src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -20,8 +24,13 @@
             @ApiResponse(code = 200, message = "Successful operation")})
     public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required = true) @RequestParam(value = "driversLicense", required = true) String driversLicense, @ApiParam(value = "surcode", required = true) @RequestParam(value = "surcode", required = true) String surcode) {
         /* TODO implement the logic as in the webmethods */
+        logger.fatal("getValidOpenDFCMCase is called");
         String integer_ = "2"; // This will be changed later to accept the int value from the ORDS
         String caseDesc = "ROUTINE - PROFESSIONAL"; // This will be changed later to accept the caseDesc value from the ORDS
+
+        if (StringUtils.isEmpty(driversLicense) || StringUtils.isEmpty(surcode)) {
+            logger.fatal("The drivers license and surcode is either empty or null ");
```

Comment:
your logging message is a bit misleading. your if statement has two objects in an OR, and your message uses an AND. this may need to be fixed?

should we break this into an if/elseif for two separate fatal messages? is there

Reply:
removed the null check as it may change when more webmethod logic comes in

Reply:
I would just remove the checks for empty parameters for now. We will have to work together on the validation requirements.

Reply:
yes removed as part of commit https://github.com/bcgov/jag-dps/pull/42/commits/b2f0c627290cfa5005526af784b578da6fa2400c